libs/math/include/math/distance/manhattan.hpp

```diff
@@ -30,15 +30,14 @@ template <typename ArrayType>
 inline typename ArrayType::Type Manhattan(ArrayType const &a, ArrayType const &b)
 {
   assert(a.size() == b.size());
-  using Type     = typename ArrayType::Type;
-  using SizeType = typename ArrayType::SizeType;
+  using Type = typename ArrayType::Type;
 
-  Type     result = 0;
-  SizeType count  = 0;
+  Type result{0};
+  auto b_ptr = b.begin();
```

Comment:
b is an iterator - it's slightly confusing to name it pointer. b_it would be fine.
Also, would be better to use cbegin() if possible.